README.md

```diff
@@ -20,6 +20,7 @@ Command-line tools for Ringpop
     status       Status of members in ring
     partitions   Show partition information of a ring
     top          General membership information
+    reap         Remove the faulty nodes from this nodes membership
```

Comment:
yeah, I'm not 100% happy with the name, but it was the best I could come up with since the feature is called 'reaping faulty nodes'. Any suggestions on a better name for the ringpop admin command?